clang/unittests/Support/TimeProfilerTest.cpp

```diff
@@ -209,7 +208,7 @@ constexpr int slow_init_list[] = {1, 1, 2, 3, 5, 8, 13, 21}; // 25th line
   ASSERT_TRUE(compileFromString(Code, "-std=c++20", "test.cc"));
   std::string Json = teardownProfiler();
   ASSERT_EQ(R"(
-Frontend
+Frontend (, test.cc)
```

Comment:
NIT: a simple check in the test output code is probably a better trade-off than descreased readability because of a weird-looking`(,`.

Reply:
Removed extraneous ` ,`